vlasovsolver/cpu_acc_transform.cpp

```diff
@@ -46,11 +47,21 @@ void updateAccelerationMaxdt(
    const Real Bz = spatial_cell->parameters[CellParams::BGBZVOL]+spatial_cell->parameters[CellParams::PERBZVOL];
    const Eigen::Matrix<Real,3,1> B(Bx,By,Bz);
    const Real B_mag = B.norm() + 1e-30;      
-   const Real gyro_period = 2 * M_PI * getObjectWrapper().particleSpecies[popID].mass
-      / (getObjectWrapper().particleSpecies[popID].charge * B_mag);
+   const Real gyro_period = fabs(2 * M_PI * getObjectWrapper().particleSpecies[popID].mass
+				 / (getObjectWrapper().particleSpecies[popID].charge * B_mag));
 
    // Set maximum timestep limit for this cell, based on a maximum allowed rotation angle
-   spatial_cell->set_max_v_dt(popID,fabs(gyro_period)*(P::maxSlAccelerationRotation/360.0));
+   spatial_cell->set_max_v_dt(popID,gyro_period*(P::maxSlAccelerationRotation/360.0));
+   
+   // Constrain Vlasov solver with plasma frequency?
+   if (P::ResolvePlasmaPeriod) {
+     Real rho = 1.e-10 > spatial_cell->get_population(popID).RHO ? 1.e-10 : spatial_cell->get_population(popID).RHO;
```

Comment:
Whose sleeve did this value come from? Is it safe to have this epsilon hard-coded here?

Reply:
Especially since there seems to be a different epsilon below...

Reply:
Consolidated epsilon handling in the next version.